technic/machines/register/cables.lua

```diff
@@ -103,7 +103,9 @@ function technic.register_cable_plate(nodename, data)
 				if (crtl.aux1 or crtl.sneak) and not (crtl.aux1 and crtl.sneak) and index ~= 0 then
 					local fine_pointed = minetest.pointed_thing_to_face_pos(placer, pointed_thing)
 					fine_pointed = vector.subtract(fine_pointed, pointed_thing.above)
-					fine_pointed[xyz[index < 0 and -index or index]] = nil
+					index = index < 0 and -index or index
+					index = (index-1)%3+1
```

Comment:
Comment could possibly be
```suggestion
					-- Normalize direction to prevent `xyz` index overflow with oversize nodes
					index = (index-1)%3+1
```